pwndbg/commands/elf.py

```diff
@@ -95,11 +95,11 @@ def print_symbols_in_section(section_name, filter_text="") -> None:
 
 def get_symbols_in_region(start, end, filter_text=""):
     symbols = []
-    ptr_size = pwndbg.gdblib.typeinfo.pvoid.sizeof
+    ptr_size = pwndbg.aglib.typeinfo.pvoid.sizeof
     addr = start
     while addr < end:
-        name = pwndbg.gdblib.symbol.get(addr, gdb_only=True)
-        if name != "" and "+" not in name and filter_text in name:
+        name = pwndbg.dbg.selected_inferior().symbol_name_at_address(addr)
+        if name and name != "" and "+" not in name and filter_text in name:
```

Comment:
The `if name` part does the same as if: `if name is not None and len(name)>0` so the `name != ""` is redundant now.
```suggestion
        if name and "+" not in name and filter_text in name:
```